lms/static/sass/base/_layouts.scss

```diff
@@ -0,0 +1,104 @@
+// base layout styles to support early responsive lms
+// may be discarded later once sass breakpoints are wired in
+
+body.view-incourse {
+  background-color: $body-bg;
+
+  .window-wrap {
+    min-width: 760px;
+  }
+
+  header.global,
+  header.global.slim {
+    width: auto;
+
+    .nav-wrapper {
+      min-width: auto;
+      padding-right: 2%;
+      padding-left: 2%;
+    }
+  }
+
+  .wrapper-course-material,
+  .wrapper-preview-menu {
+    padding: 0;
+  }
+
+  .wrapper-course-material .course-material,
+  .wrapper-preview-menu .preview-menu  {
+    padding: 15px 2%;
+    width: auto;
+  }
+
+  .wrapper-course-material .course-material .course-tabs {
+    padding: 0;
+  }
+
+  .container {
+    max-width: none;
+    min-width: initial;
+    width: auto;
+    padding: 0 2%;
+  }
+
+  .course-wrapper,
+  .profile-wrapper {
+    max-width: 1180px;
```

Comment:
Because of the scattered implementation of responsive, I decided to keep this scoped to this specific use and kept it hardcoded until we can make things more universal/variable-friendly.